contrib/requirements.txt

```diff
@@ -0,0 +1,6 @@
+pytest
+pytest-env
+pytest-xdist
+
+# Timm
+git+https://github.com/rwightman/pytorch-image-models.git#egg=timm
```

Comment:
Timm already installs a huggingface_hub version I imagine, so how would that conflict with the pip install . from before?

Reply:
Yep, huggingface_hub has to be installed in last. Thanks for review :+1: